src/spauna.ts

```diff
@@ -1,24 +1,79 @@
 // See https://stackoverflow.com/a/74112582
 /// <reference lib="dom" />
+import { readFile } from "node:fs/promises";
+import path from "node:path";
+import { pathToFileURL } from "node:url";
 
 export async function runCommand(
   commandName: string,
   configLocation: string,
   args: object,
 ) {
   // Step 1: Fetch Config
+  const config = await fetchConfig(
+    configLocation,
+    pathToFileURL(process.cwd() + "/"),
+  );
+
+  console.log(config);
```

Comment:
Is this meant to be in here? If so, it might be nice to provide some user-friendly context, e.g., `spauna running with config:` or sth.

Reply:
It's helpful for my manual testing, but will be removed as the implementation progresses.

Reply:
That's what I figured. I sometimes forget about such things, so I wanted to flag it just in case.